config/inovelli/lzw60.xml

```diff
@@ -0,0 +1,131 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/031E:000d:0001</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/inovelli/lzw31-sn.png</MetaDataItem>
```

Comment:
Eh. That doesn't seem like the right image.

Reply:
Ahh good point, I just added the correct one!